user/api_test.go

```diff
@@ -51,6 +54,14 @@ var (
 		Salt:               "a mineral substance composed primarily of sodium chloride",
 		VerificationSecret: "",
 		ClinicDemoUserID:   "00000000",
+		Marketo: marketo.Config{
```

Comment:
Is this used?

Reply:
in the ApiConfig struct it has a marketo key, so I added those fake marketo keys and values to the  fake apiconfig